src/components/StyleGuide/Iconography.mdx

```diff
@@ -31,16 +29,20 @@ The graphic of the `Icon` component is handled through the `name` prop. To see
 all graphic options for `Icon`, please refer to the [Icon](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/docs/components-media-icons-icon--icon-with-controls)
 Storybook page.
 
-<Canvas>
-  <DSProvider>
-    <Stack spacing="s" direction="row">
-      <Icon decorative size="xxxlarge" name="check" />
-      <Icon decorative size="xxxlarge" name="close" />
-      <Icon decorative size="xxxlarge" name="errorOutline" />
-      <Icon decorative size="xxxlarge" name="speakerNotes" />
-    </Stack>
-  </DSProvider>
-</Canvas>
+<Box
+  border="1px"
+  borderColor="ui.border.default"
+  borderRadius="5px"
+  mb="m"
+  p="s"
+>
+  <Stack spacing="s" direction="row">
+    <Icon decorative size="xxxlarge" name="check" />
+    <Icon decorative size="xxxlarge" name="close" />
+    <Icon decorative size="xxxlarge" name="errorOutline" />
+    <Icon decorative size="xxxlarge" name="speakerNotes" />
+  </Stack>
+</Box>
 
 | Values for Examples Above |
```

Comment:
I just realized that the nice table formatting is gone.  Do you know we can get that back?  Do we need to add our own styles?